rfc6962-bis.xml

```diff
@@ -1468,10 +1465,10 @@ but it is expected there will be a variety.
           <t>
             <list style="symbols">
               <t>
-                In the case that use of TLS with a valid certificate is mandated by explicit security policy, application protocol specification, or other means, the TLS client MUST refuse the connection.
+                In the case that use of TLS with a valid certificate is mandated by explicit security policy, application protocol specification, or other means, the TLS client SHOULD refuse the connection.
```

Comment:
> How about we change the 2 occurrences of "(the) use of TLS with a valid certificate" to "compliance" ?

I don't think that makes sense, because then it's saying that if compliance is optional (e.g. there's no TLS feature extension), clients MUST NOT treat the certificate as valid if it's non-compliant.

How about making the two bullet points conditional on "If CT compliance is required"?  I just pushed a new commit with proposed wording.